via/migrations/versions/d4e3e1bf95eb_add_the_video_table.py

```diff
@@ -0,0 +1,32 @@
+"""Add the video table.
+
+Revision ID: d4e3e1bf95eb
+Revises:
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "d4e3e1bf95eb"
+down_revision = "9a37efe13a91"
+
+
+def upgrade() -> None:
+    op.create_table(
+        "video",
+        sa.Column(
+            "created", sa.DateTime(), server_default=sa.text("now()"), nullable=False
+        ),
+        sa.Column(
+            "updated", sa.DateTime(), server_default=sa.text("now()"), nullable=False
+        ),
+        sa.Column("id", sa.Integer(), autoincrement=True, nullable=False),
```

Comment:
Same as with the other migration, we could move this to the top to reflect the mixins.

Reply:
:+1: Done